src/usdm-client.ts

```diff
@@ -156,6 +166,10 @@ export class USDMClient extends BaseRestClient {
     return this.get('fapi/v1/fundingRate', params);
   }
 
+  getFundingRateInfo(): Promise<any[]> {
```

Comment:
Does this have the same return type as the coinm client? If so, move the interface to shared (or import from coinm to reduce repetition).